### ISSUE TITLE
Add ability to define custom path and filename for config file

### ISSUE THREAD

**Issue by overag3** [leeej84/PowerScale#22]
It would be very useful if we could specify the path and filename of configuration file (config.xml) as parameter in these scripts :
- Create Config File.ps1
- and Decision Making.ps1

Regards

**Comment by leeej84** [comment]
No problems. Whats the driver for the request? Multiple instances?

How would you expect the Dashboard to be managed in this scenario?

**Comment by overag3** [comment]
Yes, we have a various delivery groups with different needs and we would be very interested to be able to specify the config file.
I didn't think about the dashboard at all ! I don't know... By adding a prefix on dashboard filename ?

**Comment by Pistoletjes** [comment]
what about simply having multiple copies (=folders) of the tool running,
each with its own configuration file?

**Comment by leeej84** [comment]
I'd recommend the multiple folder approach for now. Specifying the config file would mean a lot of changes right now. In this scenario, you'll also get a dashboard per location too.

**Comment by overag3** [comment]
no problem, i understand.
Do you want to close this issue or you want to keep this open ?

**Comment by leeej84** [comment]
Let's leave it open for now, I will try and think about it and see if can find a good way of doing it.
